public/app/plugins/panel/logs/LogsPanel.tsx

```diff
@@ -24,7 +24,10 @@ export const LogsPanel: React.FunctionComponent<LogsPanelProps> = ({
   },
   title,
 }) => {
-  const style = useStyles2(getStyles(title));
+  const isAscending = sortOrder === LogsSortOrder.Ascending;
+  const style = useStyles2(getStyles(title, isAscending));
+  const [scrollTop, setScrollTop] = useState(0);
+  const logsContainerRef = useRef<HTMLDivElement>() as React.RefObject<HTMLDivElement>;
```

Comment:
```suggestion
  const logsContainerRef = useRef<HTMLDivElement>(null);
```

nitpick: i think this can be written in a simpler way, could this be ok?

Reply:
Updated this in https://github.com/grafana/grafana/pull/37634/commits/dc971a8361e56dc977e580860bdc3f4fe4d2f470 plus I've tackled incorrect offset when refreshed.